tests/src/FunctionalJavascript/NexxTestTrait.php

```diff
@@ -257,7 +258,7 @@ protected function postVideoData($data) {
       ],
     ]);
 
-    $responseBody = \GuzzleHttp\json_decode($response->getBody()->getContents());
```

Comment:
This line is nicer than the new line imo. If this is because of a code quality scanner. It's wrong and probably worth disabling for this line. The new line makes you think you're going to get normal PHP behaviour but you're not.

Reply:
Can rector skip this line?